<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWin</class>
 <widget class="QDialog" name="FilterWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilterWin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>The number of points to use for mean distance estimation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="nr_k">
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>The standard deviation multiplier</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="stddev_mult">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="yesBtn">
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
